components/raftstore/src/store/worker/refresh_config.rs

```diff
@@ -0,0 +1,230 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+use crate::store::fsm::apply::{ApplyFsm, ControlFsm};
+use crate::store::fsm::store::StoreFsm;
+use crate::store::fsm::PeerFsm;
+use batch_system::{BatchRouter, Fsm, FsmTypes, HandlerBuilder, Poller, PoolState, Priority};
+use file_system::{set_io_type, IOType};
+use std::fmt::{self, Display, Formatter};
+use std::sync::atomic::Ordering;
+use std::thread;
+use tikv_util::worker::Runnable;
+use tikv_util::{debug, error, info, safe_panic, thd_name};
+
+pub struct PoolController<N: Fsm, C: Fsm, H: HandlerBuilder<N, C>> {
+    pub router: BatchRouter<N, C>,
+    pub state: PoolState<N, C, H>,
+}
+
+impl<N, C, H> PoolController<N, C, H>
+where
+    N: Fsm,
+    C: Fsm,
+    H: HandlerBuilder<N, C>,
+{
+    #[allow(dead_code)]
+    pub fn new(router: BatchRouter<N, C>, state: PoolState<N, C, H>) -> Self {
+        PoolController { router, state }
+    }
+}
+
+impl<N, C, H> PoolController<N, C, H>
+where
+    N: Fsm + std::marker::Send + 'static,
+    C: Fsm + std::marker::Send + 'static,
+    H: HandlerBuilder<N, C>,
+{
+    pub fn resize_to(&mut self, size: usize) -> (bool, usize) {
```

Comment:
How about using `iusize` as the return value type?

Reply:
Fixed, thanks! I removed this method, and use `match` and `std::cmp` instread.